lib/src/interop/storage_interop.dart

```diff
@@ -76,7 +76,6 @@ class FullMetadataJsImpl extends UploadMetadataJsImpl {
 @anonymous
 class UploadMetadataJsImpl extends SettableMetadataJsImpl {
   external String get md5Hash;
-  external void set md5Hash(String s);
```

Comment:
What's weird – I have *zero* clue what it's used for.

If you set it to something weird, it's not like it blocks the upload or anything...